samples/nfc/nfc_hello/README.txt

```diff
@@ -15,10 +15,14 @@ To test the serial line routines, open a terminal window and type:
 Open another terminal window and type:
 
 For QEMU x86:
-	make pristine && make run
+	mkdir build; cd build
+	cmake -DBOARD=qemu_x86 ..
+	make run
```

Comment:
Why not use zephyr-app-commands?

Reply:
this is not a RST file, it is plain text

Reply:
Ok, so why not make it an RST file?

Reply:
backlog, but this should be done in some other patch, I just noticed those old calls and figured I fix them, remaining TXT->RST need to be done at some point though.